codalab/rest/account.py

```diff
@@ -75,8 +76,23 @@ def do_signup():
     last_name = request.forms.get('last_name')
     password = request.forms.get('password')
     affiliation = request.forms.get('affiliation')
+    token = request.forms.get('token')
```

Comment:
Let's add a test called account_test.py that tests the signup endpoint (let's structure it similarly to https://github.com/codalab/codalab-worksheets/blob/master/tests/unit/rest/user_test.py). We can mock the response from the reCAPTCHA API -- this would just ensure we have this endpoint covered in our tests, which is good since we're changing it now.

Reply:
I have added the tests for the signup endpoint. We don't need mock since the default secret key will just accept any token (as long as something is given). I am just directly calling the backend API for the tests.

Reply:
Ideally, we would mock so that we don't have to hit the network for these tests. See if you can do so, but maybe if it takes too much work we can just keep it as it is.